frontend/views/components/InviteLinkToCopy.vue

```diff
@@ -0,0 +1,134 @@
+<template lang='pug'>
+component.c-wrapper(
+  :is='tag'
+  @click='copyToClipboard'
+  data-test='invitationLink'
+)
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='inviteLink'
+  )
+  span.link.has-ellipsis.c-link {{ inviteLink }}
+  button.is-icon-small.has-background.c-copy-button(:aria-label='L("Copy link")')
+    i.icon-copy.is-regular
+  transition(name='fade')
+    i18n.c-tooltip(
+      v-if='ephemeral.isTooltipActive'
+      :style='ephemeral.tooltipPosition'
+      key='test'
+      tag='p'
+      v-append-to-body=''
+    ) Copied to clipboard!
+</template>
+
+<script>
+export default {
+  name: 'InviteLinkToCopy',
```

Comment:
The way you did this component is so agnostic (👏 ) that it doesn't make sense to call it "InviteLinkToCopy". It can take any type of link to copy, so I would just call it `LinkToCopy` and rename the prop `inviteLink` to `link`.

Reply:
I agree. Changed to `LinkToCopy.vue`.